city-infrastructure-platform/urls.py

```diff
@@ -1,9 +1,21 @@
 from django.conf import settings
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
+from rest_framework import routers
+
+from traffic_control import api as traffic_control_api
+from users import api as users_api
+
+router = routers.DefaultRouter()
+router.register("users", users_api.UserViewSet)
+router.register("groups", users_api.GroupViewSet)
+router.register(
+    "trafficsignplans", traffic_control_api.TrafficSignPlanViewSet,
```

Comment:
Would it be more readable to use something like `traffic-sign-plans`?

Reply:
Updated to use `traffic-sign-plans`